controller/src/conf/controller.config.properties

```diff
@@ -68,6 +68,13 @@ controller.retention.frequency.minutes=${RETENTION_FREQUENCY_MINUTES}
 controller.retention.bucket.count=${BUCKET_COUNT}
 controller.retention.thread.count=${RETENTION_THREAD_POOL_SIZE}
 
+controller.retention.type=TIME
```

Comment:
Time -> ${RETENTION_TYPE}. 
Also we can add one comment indicating that  **controller.retention.type** value can either be TIME or SIZE.

Reply:
Done